tests/e2e/run-scanner-v4.bats

```diff
@@ -340,6 +339,118 @@ teardown_file() {
     verify_scannerV4_indexer_deployed "${CUSTOM_SENSOR_NAMESPACE}"
 }
 
+@test "[Operator] Upgrade multi-namespace installation" {
+    if [[ "${ORCHESTRATOR_FLAVOR:-}" != "openshift" ]]; then
+        skip "This test is currently only supported on OpenShift"
+    fi
+    if [[ "${ENABLE_OPERATOR_TESTS:-}" != "true" ]]; then
+        skip "Operator tests disabled. Set ENABLE_OPERATOR_TESTS=true to enable them."
+    fi
+
+    # shellcheck disable=SC2030,SC2031
+    export DEPLOY_STACKROX_VIA_OPERATOR="true"
+    # shellcheck disable=SC2030,SC2031
+    export SENSOR_SCANNER_SUPPORT=true
+
+    # Install old version of the operator & deploy StackRox.
+    (
+      VERSION="${OPERATOR_VERSION_TAG}" make -C operator deploy-previous-via-olm
+      #  cd operator
+      #  ./hack/olm-operator-install.sh stackrox-operator quay.io/rhacs-eng/stackrox-operator 4.3.0 4.3.0
+    )
+    ROX_SCANNER_V4="false" _deploy_stackrox "" "${CUSTOM_CENTRAL_NAMESPACE}" "${CUSTOM_SENSOR_NAMESPACE}"
+
+    verify_scannerV2_deployed "${CUSTOM_CENTRAL_NAMESPACE}"
+    verify_scannerV2_deployed "${CUSTOM_SENSOR_NAMESPACE}"
+
+    # Upgrade operator
+    info "Upgrading StackRox Operator to version ${OPERATOR_VERSION_TAG}..."
+    VERSION="${OPERATOR_VERSION_TAG}" make -C operator upgrade-via-olm
+    info "Waiting for rhacs-operator pods to be ready"
+    "${ORCH_CMD}" -n stackrox-operator wait  --for=condition=Ready pods -l app=rhacs-operator
+
+    verify_scannerV2_deployed "${CUSTOM_CENTRAL_NAMESPACE}"
+    verify_scannerV2_deployed "${CUSTOM_SENSOR_NAMESPACE}"
+    verify_no_scannerV4_deployed "${CUSTOM_CENTRAL_NAMESPACE}"
+    verify_no_scannerV4_indexer_deployed "${CUSTOM_SENSOR_NAMESPACE}"
+
+    wait_until_central_validation_webhook_is_ready "${CUSTOM_CENTRAL_NAMESPACE}"
+
+    # Enable Scanner V4 on central side.
+    info "Patching Central"
+    "${ORCH_CMD}" -n "${CUSTOM_CENTRAL_NAMESPACE}" \
+      patch Central stackrox-central-services --type=merge --patch-file=<(cat <<EOT
+spec:
```

Comment:
Could we also replace this two patches with using the newly added CR files, or do I overlook the difference here?

Reply:
Well, this is *only* adding a Scanner V4 config, the persisted CRs contain much more, especially envsubst templating, for which there is no need to replicate here.

Reply:
Ok, sounds good.